Tools/jit/_stencils.py

```diff
@@ -181,6 +182,9 @@ class Stencil:
     body: bytearray = dataclasses.field(default_factory=bytearray, init=False)
     holes: list[Hole] = dataclasses.field(default_factory=list, init=False)
     disassembly: list[str] = dataclasses.field(default_factory=list, init=False)
+    aarch64_trampolines: dict[str | None, int] = dataclasses.field(
```

Comment:
Maybe just rename this to "trampolines", since we may use short jumps on more platforms in the future:

```suggestion
    trampolines: dict[str | None, int] = dataclasses.field(
```